releaseNotes/Microsoft.Featuremanagement.Telemetry.ApplicationInsights.md

```diff
@@ -2,6 +2,21 @@
 
 [Source code][source_code] | [Package (NuGet)][package] | [Samples][samples] | [Product documentation][docs]
 
+## 4.0.0 - September 24, 2024
+
+### Enhancements
+
+#### Application Insights
+
+Added support for Application Insights telemetry. To publish feature flag evaluation data and tag outgoing events with targeting information, register the Application Insights telemetry publisher as shown below.
+
+```csharp
+builder.Services.AddFeatureManagement()
+    .AddApplicationInsightsTelemetry();
+```
+
+For more details on Application Insights Publishing, see [here](https://learn.microsoft.com/en-us/azure/azure-app-configuration/feature-management-dotnet-reference#application-insights-telemetry-publisher)
```

Comment:
Does this link work for you consistently. It seemed to work for me once then break when I viewed it again as if it depended upon my previous tab state on the page.

Reply:
Interesting, I thought it was just me having that problem... It always a problem even if you include the `?pivots=preview-version`. I suspect the problem will go away once the pivots are gone.